src/modules/samples/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-add_subdirectory(cpp)
\ No newline at end of file
+add_subdirectory(cpp)
+add_subdirectory(rust)
```

Comment:
>rust [](http://example.com/codeflow?start=17&length=4)

All modules samples here must follow the pattern show in this C++ sample. Basically showing how to implement all possible MIM cases in a module. If the C++ sample is very useful like this (we got such feedback), the Rust one will be seen significantly a lot more useful for developers.